examples/boise_river_basin/brb_wy2017.ini

```diff
@@ -1,30 +1,25 @@
 ################################################################################
-#
```

Comment:
What are these examples still used for?

Reply:
Nothing. I was just trying to limit the scope of the work since it was already pretty far reaching. I can remove them if you would like.

Reply:
Leave them in and we'll address this with a larger testing suite at some point.